src/cc_catalog_airflow/dags/provider_api_scripts/brooklyn_museum.py

```diff
@@ -36,156 +36,159 @@
     'skip': 0
```

Comment:
in brooklyn museum offset should be used instead of skip

Reply:
okay, I am working on this.